tests/components/rfxtrx/test_cover.py

```diff
@@ -114,35 +122,19 @@ async def test_discover_covers(hass, rfxtrx):
     )
     await hass.async_block_till_done()
 
-    event = rfxtrx_core.get_rfx_object("0a140002f38cae010f0070")
-    event.data = bytearray(
-        [0x0A, 0x14, 0x00, 0x02, 0xF3, 0x8C, 0xAE, 0x01, 0x0F, 0x00, 0x70]
-    )
-
-    await _signal_event(hass, event)
-    assert 1 == len(rfxtrx_core.RFX_DEVICES)
-
-    event = rfxtrx_core.get_rfx_object("0a1400adf394ab020e0060")
-    event.data = bytearray(
-        [0x0A, 0x14, 0x00, 0xAD, 0xF3, 0x94, 0xAB, 0x02, 0x0E, 0x00, 0x60]
-    )
+    await _signal_event(hass, "0a140002f38cae010f0070")
+    assert len(hass.states.async_all()) == 1
 
-    await _signal_event(hass, event)
-    assert 2 == len(rfxtrx_core.RFX_DEVICES)
+    await _signal_event(hass, "0a1400adf394ab020e0060")
+    assert len(hass.states.async_all()) == 2
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)